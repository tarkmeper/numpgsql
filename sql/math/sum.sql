SELECT '{1,2}'::integer[] + '{3,4}'::integer[];

-- One operand
SELECT '{}'::integer[] + 3;
SELECT '{1,2,5}'::integer[] + 3;
SELECT '{{1,2},{3,4}}'::integer[] + 3;
SELECT '{1,2}'::integer[] + 3.5;

