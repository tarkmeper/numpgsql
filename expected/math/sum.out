SELECT '{1,2}'::integer[] + '{3,4}'::integer[];
 ?column? 
----------
 {4,6}
(1 row)

-- One operand
SELECT '{}'::integer[] + 3;
 ?column? 
----------
 {}
(1 row)

SELECT '{1,2,5}'::integer[] + 3;
 ?column? 
----------
 {4,5,8}
(1 row)

SELECT '{{1,2},{3,4}}'::integer[] + 3;
   ?column?    
---------------
 {{4,5},{6,7}}
(1 row)

SELECT '{1,2}'::integer[] + 3.5;
ERROR:  operator does not exist: integer[] + numeric
LINE 1: SELECT '{1,2}'::integer[] + 3.5;
                                  ^
HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.
